website/docs/feature-flag-tutorials/use-cases/trunk-based-development.md

```diff
@@ -0,0 +1,222 @@
+
+## How to Implement Trunk-Based Development with Feature Flags
+
+To follow along with this tutorial, you need access to an Unleash instance to create and manage feature flags. Head over to our [Quick Start documentation](/quickstart) for options, including running locally or using an [Unleash SaaS instance](https://www.getunleash.io/pricing?).
+
+With Unleash set up, you can use your application to talk to Unleash through one of our [SDKs](/reference/sdks).
+
+In this tutorial, you will:
+
+1. [Explore many trunk-based development strategies](#choose-the-right-trunk-based-development-strategy-for-your-team)
+2. [Create feature flags for incomplete features](#create-feature-flags-for-incomplete-features)
+3. [Control feature rollouts](#control-feature-rollouts)
+4. [Simplify rollbacks with Unleash](#simplify-rollbacks)
+5. [Archive and remove feature flags](#archive-and-remove-feature-flags)
+
+## Choose the Right Trunk-Based Development Strategy for Your Team
+
+There is no one-size-fits-all approach to implementing trunk-based development. Different teams may benefit from varying strategies based on their needs, project complexity, and development culture.
+
+In this section, we'll explore the common trunk-based development strategies and their trade-offs to help you choose the best approach for your organization.
+
+### Merge Every Commit
+
+The most frequent approach to trunk-based development is merging every commit directly into the main branch. This strategy maximizes the rate of continuous integration and delivery, as changes are integrated into the trunk as soon as they are ready. Some advantages are:
+
+-   Rapid feedback loops and quicker issue resolution
+-   Reduced risk of large, complex merges
+-   Encourages small, frequent commits
+
+Some challenges with merging every commit are:
+
+-   Potential for more frequent build breaks and test failures
+-   Increased need for effective merging strategies
+-   Requirement for robust automated testing and CI/CD pipelines
+
+### Merge at Set Intervals
+
+Another common strategy is to merge changes into the trunk at regular, predefined intervals, such as daily or before the end of each work day. This approach offers a balance between frequent integration and managing work-in-progress.
+
+**Advantages**:
+
+-   Improved control over the integration process
+-   Reduced risk of breaking the build with frequent merges
+-   Easier to manage and identify integration issues
+
+**Challenges**:
+
+-   Longer feedback loops compared to merging every commit
+-   Potential for larger, more complex merges
+-   Requires discipline to adhere to the defined merge cadence
+
+### Benefits of Short-Lived Feature Branches
```

Comment:
This is 100% a personal feeling so take it for what it's worth.

I know folks often call these branches short lived feature branches but I think that completely misses the point. A feature is often cleaved up into pieces that span many short lived branches, a toggle prevents an incomplete feature being exposed before it's done